php/WC_Gateway_Blockonomics.php

```diff
@@ -194,6 +194,7 @@ public function check_blockonomics_callback()
         }
         $address = isset($_REQUEST['get_order']) ? $_REQUEST['get_order'] : "";
         if ($address) {
+            $orders[$address]['remaining_time'] = $orders[$address]['timestamp'] + 600 - time();
```

Comment:
The calculation for time remaining should still use the blockonomics timeperiod setting:
 `get_option('blockonomics_timeperiod', 10);`